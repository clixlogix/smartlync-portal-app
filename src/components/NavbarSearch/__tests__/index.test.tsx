import * as React from 'react';
import { render } from '@testing-library/react';

import { NavbarSearch } from '..';

describe('<NavbarSearch  />', () => {
    it('should match snapshot', () => {
        const loadingIndicator = render(<NavbarSearch />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
