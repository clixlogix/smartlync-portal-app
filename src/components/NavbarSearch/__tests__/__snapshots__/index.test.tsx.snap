// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarSearch  /> should match snapshot 1`] = `
<div
  class=" navSearch"
>
  <div
    class="custom-search-bar"
  >
    <div
      class="input-wrapper"
    >
      <input
        class="input"
        placeholder="Search"
        type="text"
      />
    </div>
    <div
      class="search-icon-wrapper"
    >
      <i
        class="fa fa-search"
      />
    </div>
  </div>
</div>
`;
